onnxruntime/core/optimizer/selectors_actions/actions.h

```diff
@@ -147,13 +147,13 @@ struct ReplaceWithNewFixed : public ReplaceWithNew {
         value_moves_{std::move(value_moves)} {
   }
 
+  NodeAttributes ExtraAttributes(const RuntimeState&) const override { return extra_attrs_; }
+
```

Comment:
what is the reason for making this public?

Reply:
Decorated as Protected